src/main/kotlin/no/nav/syfo/infrastructure/client/arkivporten/ArkivportenDocumentRequestDTO.kt

```diff
@@ -0,0 +1,103 @@
+package no.nav.syfo.infrastructure.client.arkivporten
+
+import com.fasterxml.jackson.annotation.JsonValue
+import java.util.UUID
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.domain.Virksomhetsnummer
+import no.nav.syfo.domain.dialogmote.MotedeltakerVarselType
+
+data class ArkivportenDocumentRequestDTO(
+    val documentId: UUID,
+    val type: DocumentType,
+    val content: ByteArray = byteArrayOf(),
+    val contentType: ContentType,
+    val orgNumber: String,
+    val title: String,
+    val summary: String,
+    val fnr: String,
+    val fullName: String,
+) {
+
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as ArkivportenDocumentRequestDTO
+
+        if (type != other.type) return false
+        if (!content.contentEquals(other.content)) return false
+        if (contentType != other.contentType) return false
+        if (orgNumber != other.orgNumber) return false
+        if (title != other.title) return false
+        if (summary != other.summary) return false
+
+        return true
+    }
+
+    override fun hashCode(): Int {
+        var result = type.hashCode()
+        result = 31 * result + content.contentHashCode()
+        result = 31 * result + contentType.hashCode()
+        result = 31 * result + orgNumber.hashCode()
+        result = 31 * result + title.hashCode()
+        result = 31 * result + summary.hashCode()
+        result = 31 * result + fnr.hashCode()
+        result = 31 * result + fullName.hashCode()
+        return result
+    }
+
+    enum class DocumentType {
+        DIALOGMOTE,
+    }
+
+    enum class ContentType(val value: String) {
+        PDF("application/pdf");
+
+        @JsonValue
+        override fun toString() = value
+    }
+
+    companion object {
+        fun createArkivportenDokument(
```

Comment:
Det er kanskje tydelig nok hvilken kontekst vi er i her? 😊
```suggestion
        fun create(
```